database/gdb/gdb_model_join.go

```diff
@@ -93,3 +93,38 @@ func (m *Model) doJoin(operator string, table ...string) *Model {
 	}
 	return model
 }
+
+// StraightJoin use STRAIGHT_JOIN
+// Refer: https://stackoverflow.com/questions/512294/when-to-use-straight-join-with-mysql
+func (m *Model) StraightJoin(table ...string) *Model {
```

Comment:
Please implement the logic in function `doJoin`, add detailed comment and corresponding unit testing case for your feature.